extensions/adobe/experience/campaign/experienceevent.example.email-send.json

```diff
@@ -59,12 +59,16 @@
     "xdm:from": "no-reply@adobe.com",
     "xdm:testEnabled": true,
     "xdm:messageClass": "continuous",
-    "xdm:templateID": 1000
+    "xdm:templateID": 1000,
+    "xdm:deliveryName": "DM200",
+    "xdm:deliveryLabel": "Birthday Wishes"
   },
   "https://ns.adobe.com/experience/campaign/marketingCampaign": {
-    "xdm:id": 100
+    "xdm:id": 100,
```

Comment:
ID isn't a number - it's a string.

Reply:
Here,"id" attribute is an integer as per the XDM schema.

Reply:
It is custom attribute xdm:id which is of type integer but not "@id" whose type is string